app/code/Magento/Swatches/view/frontend/templates/product/layered/renderer.phtml

```diff
@@ -37,36 +38,36 @@
                             <?php $swatchImagePath = $block->getSwatchPath('swatch_image', $swatchData['swatches'][$option]['value']); ?>
                             <div class="swatch-option image <?= $block->escapeHtmlAttr($label['custom_style']) ?>"
                                  tabindex="-1"
-                                 option-type="2"
-                                 option-id="<?= $block->escapeHtmlAttr($option) ?>"
```

Comment:
What's the reason to remove escape functions from the templates? It may cause a security vulnerability without the filtration

Reply:
Hello @sidolov,

I think this changes are added after my commit.
Please check below PR Details.

https://github.com/magento/magento2/commit/50329e21a45254eb210dc8293bbc5705fd866c19#diff-f4e3f76f4cf865522b96b664b0a390ac

Please let me know should I need to add this changes in this pull request.

Thank You,
Nirav Patel

Reply:
So, does the option id was removed from the template?

Reply:
Hello @sidolov,

- option-id is already there before and after update the code.
After update the code option-id becomes data-option-id.

Please check and let me know for further changes.

Thank You,
Nirav Patel

Reply:
I see that option attributes became a data-option attributes, but why you removed `escapeUrl ` function call? 
Please, make sure that all option attributes are filtered as before.

Reply:
Hello @sidolov ,
AS you can see in my previous comment i have mentioned about the `escapeurl` changes.

This changes are applied in the file after creating this PR.

50329e2#diff-f4e3f76f4cf865522b96b664b0a390ac

I will add this changes in my pull request.

Thank You,
Nirav Patel

Reply:
In this case, merge conflicts were resolved incorrectly, you should keep remote changes along with yours.